editor/plugins/text_shader_editor.h

```diff
@@ -148,7 +148,7 @@ class TextShaderEditor : public MarginContainer {
 	ConfirmationDialog *erase_tab_confirm = nullptr;
 	ConfirmationDialog *disk_changed = nullptr;
 
-	ShaderTextEditor *shader_editor = nullptr;
+	ShaderTextEditor *code_editor = nullptr;
```

Comment:
This is the case which I said was to reduce confusion. The variable isn't exposed so I thought it would be fine to rename.

The containing class is TextShaderEditor, and is referred to by the plugin as shader_editor. This is fine, it's an editor of text shaders, so it's ultimately a shader editor. However the class also itself contains a variable called shader_editor. It also happens to be the variable I needed to expose. It would've resulted in shader_editor.get_shader_editor(). This variable's type is ShaderTextEditor, which is a subclass of CodeTextEditor. It's ultimately an editor of text. So a good renaming is to make it consistent with other classes on the script side of things, and call it a text_editor.

With this in mind, should I still move the variable rename to a different PR? I would like to at least keep the naming of the function I added (get_text_editor).

Reply:
@KoBeWi I've fixed your other reviews. For this one, could you explain to me the rationale behind shying away from renaming a private variable like this? And if we absolutely have to keep the old name for the private var, is it okay if we have a `get_code_editor()` method that returns a var called `shader_editor`?

Reply:
This change creates noise in the diff, but since there weren't other significant changes in that file, maybe it's fine in the end.